src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -66,25 +66,25 @@
 							:container="container"
 							:close-on-select="false"
 							@select="addEmoji">
-							<button type="button"
+							<Button class="emoji-picker-button"
```

Comment:
The class here has to be removed. Remember, no styling of the button from outside the button component :)